genlib/dxfp.cpp

```diff
@@ -68,27 +68,18 @@ const DxfVertex& DxfParser::getExtMax() const
    return m_region.getExtMax();
 }
 
-const DxfLayer& DxfParser::getLayerNum( const int i ) const
-{
-   return m_layers[i];
-}
-
 DxfLayer *DxfParser::getLayer( const std::string& layer_name ) // const <- removed as m_layers may be changed if DXF is poor
 {
    static DxfLayer layer;

```

Comment:
This can go. Yay! (I mean the static layer)

Reply:
If I remove the static then loading the DXF takes ages

Reply:
Solved, this needed the int member variables of the DxfLayer to be initialised otherwise it got random data and if that was a huge number then later it tried to init the SuperSpacePixels with huge sizes.